src/Ripple/Transaction.h

```diff
@@ -0,0 +1,65 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "../proto/Ripple.pb.h"
+#include "../Data.h"
+#include "Address.h"
+
+namespace TW {
+namespace Ripple {
+
+enum FieldType: int {
```

Comment:
If this is only used in C++ let's use `class enum FieldType: int` and change enum cases to just the name like `int16`

Reply:
done